obspy/io/reftek/core.py

```diff
@@ -0,0 +1,341 @@
+# -*- coding: utf-8 -*-
+"""
+REFTEK130 read support, core routines.
+"""
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+from future.builtins import *  # NOQA
+from future.utils import native_str
+
+import io
+import os
+import warnings
+
+import numpy as np
+
+from obspy import Trace, Stream, UTCDateTime
+from obspy.io.mseed.headers import clibmseed
+
+from .packet import (Packet, EHPacket, _initial_unpack_packets, PACKET_TYPES,
+                     PACKET_TYPES_IMPLEMENTED, PACKET_FINAL_DTYPE)
+
+
+NOW = UTCDateTime()
+
+
+def _is_reftek130(filename):
+    """
+    Checks whether a file is REFTEK130 format or not.
+
+    :type filename: str
+    :param filename: REFTEK130 file to be checked.
+    :rtype: bool
+    :return: ``True`` if a REFTEK130 file.
+
+    Checks if overall length of file is consistent (i.e. multiple of 1024
+    bytes) and checks for valid packet type identifiers in the first 20
+    expected packet positions.
+    """
+    if not os.path.isfile(filename):
+        return False
+    filesize = os.stat(filename).st_size
+    # check if overall file size is a multiple of 1024
+    if filesize < 1024 or filesize % 1024 != 0:
+        return False
+
+    with open(filename, 'rb') as fp:
+        # check first 20 expected packets' type header field
```

Comment:
Euuuhm.. this must've been lost during rebasing or something.. or I just dreamed that I've implemented it.. ;-)